modules/Matting/Global-Sampling-Method-for-Alpha-Matting/globalmatting.h

```diff
@@ -0,0 +1,9 @@
+#ifndef GLOBAL_MATTING_H
+#define GLOBAL_MATTING_H
+
+#include <opencv2/opencv.hpp>
+
+void expansionOfKnownRegions(cv::InputArray img, cv::InputOutputArray trimap, int niter = 9);
```

Comment:
The number of iterations here, is this always fixed? Do we want to change this value?

Reply:
This function was used in the paper Shared sampling for real time matting, and it is mentioned that the number of iterations must be around 10.

Reply:
Then I do get we want to set a default value, for example 9, but we would really like to have a setter and getter to change that value when desired.